SpatialGDKEditor/SpatialGDKEditor.uplugin

```diff
@@ -0,0 +1,30 @@
+{
+	"FileVersion" : 3,
```

Comment:
At the moment the GDK is split up into two plugins, the GDK and the toolbar. This is somewhat of a mistake, as there should only be one plugin with two modules in them. (Cleanup task here: https://improbableio.atlassian.net/browse/UNR-787)

Because of this, I don't think this new module should also be defined as a plugin. It can just stay as a module and be included in both plugins (for now). When the cleanup task comes through, we can then have 3 modules.

Reply:
So in order to achieve this, I needed to make some minor organizational changes to the existing SpatialGDK plugin's folder structure (just had to pop all of it's existing source into another subfolder). This cascaded into what ultimately came down to a single line change to a path so that a spatial binary can be found at build-time, but otherwise it looks okay.

Reply:
Once that was done and working, I could make a sibling folder to put the SpatialGDKEditor source into, and all is well.

Reply:
Ok, the SpatialGDKEditor and SpatialGDKEditorCommandlet modules have been taken out of their own plugins, and put into the SpatialGDK plugin, alongside the SpatialGDK module. Everything looks nice and orderly from here. How's it look to you?

Reply:
So I just discovered that the commandlets aren't discoverable when they're set up this way. I'm not sure if it's a bug I introduced while moving them, or if they just need to be a separate plugin. Investigating.

Reply:
So it was a bug that was introduced because of moving files around and such. Had to rename some files/etc to have Unreal auto-detect things properly.

Reply:
Ah pesky hard coded strings

Reply:
"Technically" strings, but actually class/file names.

Reply:
Pesky Unreal

Reply:
There's so much "magic" that Unreal does, where you can't rely on intellisense AT ALL (and just "need to know" certain things), it really reminds me of the main reasons I don't like working with web technologies.